### PR TITLE
gatsby/feature/homepage v5 facelift

### PR DESCRIPTION
**Pull request by Vesely**
- Facelift header, cotent box, situation
- Add white content box variant, set primary btn blue, add yellow btn
- Add main content default bg image

### CODE REVIEW

File: gatsby/src/components/content-box/content-box.tsx [hunk 1]
```diff
@@ -41,7 +43,13 @@ const ContentBox: React.FC<Props> = ({
   }, [title]);
 
   return (
-    <div className={classNames(styles.contentBox, 'row')}>
+    <div
+      className={classNames(
+        styles.contentBox,
+        styles[`contentBox--${variant}`],
```
Comment (fidelman) [inline gatsby/src/components/content-box/content-box.tsx:49]: please, make sure PurgeCSS does not remove it. https://cesko-digital.slack.com/archives/C01DVGTNRUL/p1604246550476900

File: gatsby/src/components/search-box/search-box.module.scss [hunk 2]
```diff
@@ -1,3 +1,8 @@
 .searchBox {
-  margin: 12px 0;
+  margin-top: 14px;
```
Comment (fidelman) [inline gatsby/src/components/search-box/search-box.module.scss:2]: margin: 14px 0;

File: gatsby/src/layouts/default-layout.module.scss [hunk 3]
```diff
@@ -1,3 +1,9 @@
 .wrapper {
-  background: #337dbd;
+  background: #e5e5e5;
+}
+
+.main {
+  background-image: url(/images/main-content-mobile-bg.jpg);
```
Comment (fidelman) [inline gatsby/src/layouts/default-layout.module.scss:6]: I think we will get from BE, please expose it to the component

**Review by fidelman — CHANGES_REQUESTED** [review]
great job, left a few comments

**Comment by Vesely** [comment]
@fidelman review pls

**Review by fidelman — APPROVED** [review]

**Pull request merged**